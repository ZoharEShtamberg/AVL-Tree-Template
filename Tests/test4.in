add_country 1 0
add_team 440 1 0 
get_team_strength 440
add_team 69 1 0
play_match 440 69
get_medals 1
austerity_measures 440
add_contestant 20 1 0 90
add_contestant 21 1 0 91
add_contestant 22 1 0 92
add_contestant_to_team 440 20
add_contestant_to_team 440 21
add_contestant_to_team 440 22
get_team_strength 440
play_match 440 69
get_medals 1