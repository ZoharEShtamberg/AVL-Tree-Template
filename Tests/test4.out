add_country: SUCCESS
add_team: SUCCESS
get_team_strength: SUCCESS, 0
add_team: SUCCESS
play_match: SUCCESS
get_medals: SUCCESS, 0
austerity_measures: FAILURE
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
get_team_strength: SUCCESS, 273
play_match: SUCCESS
get_medals: SUCCESS, 1